test/forge/blue/TestBlueSnippets.sol

```diff
@@ -222,6 +222,27 @@ contract TestIntegrationSnippets is BaseTest {
         assertEq(morpho.expectedSupplyAssets(marketParams, address(snippets)), 0, "supply assets");
     }
 
+    function testWithdrawAmountOrAll(uint256 amountSuplied, uint256 amountWithdrawn) public {
+        amountSuplied = bound(amountSuplied, 1, MAX_TEST_AMOUNT);
+        amountWithdrawn = bound(amountWithdrawn, 1, MAX_TEST_AMOUNT);
+        loanToken.setBalance(address(snippets), amountSuplied);
+
+        snippets.supply(marketParams, amountSuplied, address(snippets));
+        (uint256 assetsWithdrawn,) = snippets.withdrawAmountOrAll(marketParams, address(snippets), amountWithdrawn);
+
+        if (amountSuplied >= amountWithdrawn) {
```

Comment:
We could bissect this test into 2 cases:
1. where `amountSupplied` is fuzzed to be >= `amountWithdrawn`
2. the other case

Reply:
I'm fine with the current version but I'm okay to change if you want

Reply:
The benefit is each side is fuzzed more, but it's fine for snippets to bissect this way